doc/sphinx/language/cic.rst

```diff
@@ -24,7 +24,7 @@ to a type and takes the form “*for all x of type* :math:`T`, :math:`P`”. The
 “:math:`x` *of type* :math:`T`” is written “:math:`x:T`”. Informally, “:math:`x:T`” can be thought as
 “:math:`x` *belongs to* :math:`T`”.
 
-The types of types are *sorts*. Types and sorts are themselves terms
+The types of types are :gdef:`sort`\s. Types and sorts are themselves terms
```

Comment:
BTW, I think I'll add support for <> inside gdef as its' supported in :term: -- to allow for plurals, capitalization, etc.